hokusai/development.yml

```diff
@@ -1,22 +1,20 @@
----
 version: '2'
 services:
   force:
     extends:
       file: common.yml
       service: force
     environment:
-    - NODE_ENV=development
-    - REDIS_URL=redis://force-redis:6379/0
-    - PORT=5000
-    - S3_KEY
-    - S3_SECRET
-    - DEPLOY_ENV
+      - NODE_ENV=development
+      - REDIS_URL=redis://force-redis:6379/0
+      - PORT=5000
+      - S3_KEY
+      - S3_SECRET
+      - DEPLOY_ENV
     env_file: ../.env
     ports:
-    - 5000:5000
-    command: npm start
```

Comment:
`command: npm start` got removed to make it consistent with staging/prod which don't have a command and fallback to `CMD`